<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Slider</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Horizontal with tick marks (you're able to select only values represented by tick marks)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCocoaSlider" name="sliderHorizontal" native="true"/>
          </item>
          <item>
           <widget class="QLabel" name="labelSliderValue">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Circular:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCocoaSlider" name="sliderCircular" native="true"/>
          </item>
          <item>
           <widget class="QLabel" name="labelCircularValue">
            <property name="text">
             <string>s</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Vertical slider (try clicking with Option key pressed, it will move the knob by increment value of 10)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCocoaSlider" name="sliderVertical" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelVertSlider">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Button</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCocoaButton" name="buttonRounded" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonRegularSquare" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonShadowLessSquare" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonTexturedSquare" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonSmallSquare" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCocoaButton" name="buttonTexturedRounded" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonRoundRect" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonDislosure" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonRoundedDisclosure" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonCircular" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonRecessed" native="true"/>
          </item>
          <item>
           <widget class="QCocoaButton" name="buttonInline" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>340</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Discard|QDialogButtonBox::YesToAll</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Segmented">
       <attribute name="title">
        <string>Segmented Control</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Gradient Button</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>261</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item>
          <widget class="QCocoaGradientButton" name="gradientButton" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>50</y>
          <width>258</width>
          <height>214</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableWidget"/>
         </item>
         <item>
          <widget class="QCocoaGradientButton" name="gradientWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCocoaSlider</class>
   <extends>QWidget</extends>
   <header>../qcocoaslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCocoaButton</class>
   <extends>QWidget</extends>
   <header>../qcocoabutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCocoaGradientButton</class>
   <extends>QWidget</extends>
   <header>../qcocoagradientbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
